src/functions/includegraphics.js

```diff
@@ -114,7 +128,7 @@ defineFunction({
             width = calculateSize(group.width, options);
         }
 
-        const style = {height: height + depth + "em"};
+        const style: CssStyle = {height: height + depth + "em"};
         if (width > 0) {
             style.width = width + "em";
```

Comment:
The flow errors here stemmed from the fact that when you create an object literal, flow considers that to be its type and complains about adding new properties after the fact.  To get around this you have to specify the type explicitly.